q_lib/include/q/synth/noise.hpp

```diff
@@ -0,0 +1,103 @@
+/*=============================================================================
+   Copyright (c) 2014-2021 Joel de Guzman. All rights reserved.
```

Comment:
Feel free to add your name in there. You wrote it :-)
Just don't forget to attribute the code you got this from (e.g. comments before the class).